### PR TITLE
Add the the location of the lists of available versions to the README

### PR DESCRIPTION
**Pull request by axelson**
This will help someone that doesn't know which versions are valid to include. Also added a link to Elixir and Erlang compatibility since that is useful when determining the versions to use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,12 @@
 #### Version support
 
 * Erlang - Prebuilt packages (17.5, 17.4, etc)
-* Elixir - Prebuilt releases (1.0.4, 1.0.3, etc) or prebuilt branches (master, stable, etc)
+  * The full list of prebuilt packages can be found here: https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/otp-versions
```
Comment (ericmj) [inline README.md]: Can you add a note that if a version is missing you can send a PR that adds it.

Comment (axelson) [reply README.md]: Added!

**Comment by ericmj** [comment]
Thank you!

**Comment by axelson** [comment]
Thank you for merging it!

**Pull request merged**